centreon/packaging/centreon-web.yaml

```diff
@@ -186,6 +186,13 @@ contents:
       group: "centreon"
       mode: 0755
 
+  - src: "../bin/migrateCredentials.php"
+    dst: "/usr/share/centreon/bin/migrateCredentials.php"
+    file_info:
+      owner: "root"
```

Comment:
the PR description says the script is intended to be run as root only, shouldn t the permissions reflect this ?